client/src/components/dashboard/Add.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+// need connect function to be able to connect to store from Provider
+import {connect} from 'react-redux';
+
+import {dashDefault} from '../../actions/dashActions';
+
+import moment from 'moment';
+
+class Container extends React.Component {
+    constructor(props) {
+        super(props);
+        // this.setDashDefault = this.setDashDefault.bind(this);
+    }
+
+    submitForm = (e) => {
+        e.preventDefault();
+        const formData = new FormData(document.getElementById('form-add'));
```

Comment:
I think it's better practice to use `refs` instead.
https://stackoverflow.com/questions/37273876/reactjs-this-refs-vs-document-getelementbyid

Reply:
I feel like using refs is unnecessary in this case..because it just complicates things.

https://reactjs.org/docs/refs-and-the-dom.html

the implementation of refs before was very straightforward - you'd need just a string and a ref attribute, but it has been deprecated since or discouraged to use (see Legacy API: String Refs). I feel like getElementById does the job of just selecting an element.